docs/components/TemplatesSystem/features/customTags/gimmelist.rst

```diff
@@ -71,6 +71,16 @@ To list articles from the ``Forbes`` source but without an ``AAP`` source you ca
 It will then list all articles **with** source ``Forbes`` and **without** ``AAP``.
 
 
+Listing article's custom fields:
+
+.. code-block:: twig
+
+    {% gimmelist article from articles %}
+        {{ article.title }}
+        {{ article.extra['my-custom-field'] }}
```

Comment:
how about getter for extra fields, like: {{ article.getExtra('my-custom-field') }}?

Reply:
I can add it, but I would stick to standard twig syntax because we would then need to maintain two ways of accessing these fields

Reply:
I see that we don't have option to add getter to meta - so for now it's not possible. Let's leave it as it is.